lib/core/versions/latest/Did.ts

```diff
@@ -53,48 +55,45 @@ export default class Did {
     }
 
     this.shortForm = didMethodName + this.uniqueSuffix;
+  }
+
+  /**
+   * Parses the input string as Sidetree DID.
+   * @param did Short or long-form DID string.
```

Comment:
Param doc is not matching the actual function parameters.

Reply:
Thanks, fixed.